domain-logic/pipeline-logic/lib/index.js

```diff
@@ -15,6 +15,8 @@ var ERRORS = {
   NOT_ACCESSIBLE: 1
 }
 
+var TIMEOUTERR = 'ETIMEDOUT'; 
```

Comment:
ERRORS are errors thrown directly by this class.

TIMEOUTERR is an error caught by this module so it can give a better error message, but it is thrown as is to the upper level.

Mor Shemesh  | Partner Catalyst Team

From: Ami Turgman [mailto:notifications@github.com]
Sent: Tuesday, April 12, 2016 3:54 PM
To: CatalystCode/corpus-to-graph-genomics <corpus-to-graph-genomics@noreply.github.com>
Cc: Mor Shemesh <morshe@microsoft.com>
Subject: Re: [CatalystCode/corpus-to-graph-genomics] add rescoring test (#2)


In domain-logic/pipeline-logic/lib/index.js<https://github.com/CatalystCode/corpus-to-graph-genomics/pull/2#discussion_r59369737>:

> @@ -15,6 +15,8 @@ var ERRORS = {

>    NOT_ACCESSIBLE: 1

>  }

>

> +var TIMEOUTERR = 'ETIMEDOUT';

why not adding this error to the ERRORS enum one line earlier?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/CatalystCode/corpus-to-graph-genomics/pull/2/files/70c4de45b1520409f5f3a50b022b40eed27349d7#r59369737>